frontend/src/components/JobCard/JobCard.css

```diff
@@ -41,6 +39,7 @@
 .card-title {
   margin-bottom: 0.5rem;
 }
+
 .job-descp {
   margin-bottom: 0.5rem;
```

Comment:
Hey! Please consider adding `height` prop to each part (title, desc and footer) in order to prevent breaking UI when texts have different lengths.
![Screenshot 2023-10-09 at 16 06 45](https://github.com/HackYourFuture-CPH/class-25-ironside/assets/48878547/9f376a95-a7d3-441e-a82f-70109c0ebbe1)